src/commands/blobs/blobs.ts

```diff
@@ -33,7 +33,7 @@ export const createBlobsCommand = (program: BaseCommand) => {
     )
     .argument('<store>', 'Name of the store')
     .argument('<key>', 'Object key')
-    .option('-o, --output <path>', 'Defines the filesystem path where the blob data should be persisted')
+    .option('-O, --output <path>', 'Defines the filesystem path where the blob data should be persisted')
```

Comment:
`-o` is already used in the base command for `--offline`, so this was being quietly ignored (I tested it locally).

This is fine to be considered a patch change since it has never worked.

Caught by changes in commander upgrade: https://github.com/tj/commander.js/pull/2055.

Reply:
Sort of a bummer--I think `-o` for output is a somewhat universal convention. Maybe something for us to change in future release? I think I'd opt for making `--offline` a long option only.